### ISSUE TITLE
3Blue1Brown — Neural Networks / Deep Learning (chapters 1–4)

### ISSUE THREAD

**Issue by atticusrussell** [atticusrussell/ballbot#206]
**Neural Networks / Deep Learning** — 3Blue1Brown, chapters 1–4. What a neural network is, gradient descent, backpropagation.

- [ ] [Ch 1 — But what is a neural network?](https://youtu.be/aircAruvnKk)
- [ ] [Ch 2 — Gradient descent, how neural networks learn](https://youtu.be/IHZwWFHWa-w)
- [ ] [Ch 3 — Backpropagation, intuitively](https://youtu.be/Ilg3gGewQ5U)
- [ ] [Ch 4 — Backpropagation calculus](https://youtu.be/tIeHLnjs5U8)
